### ISSUE TITLE
Pod install failed on version 3.4.0

### ISSUE THREAD

**Issue by kockok** [airbnb/lottie-ios#1652]
## Which Version of Lottie are you using?

Lottie 3.4.0, react-native 0.69

## Expected Behavior 
pod install successful.

## Actual Behavior
```
react_native_pods_utils/script_phases.rb:51: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.
[!] CocoaPods could not find compatible versions for pod "lottie-ios":
  In snapshot (Podfile.lock):
    lottie-ios (from `../node_modules/lottie-ios`)

  In Podfile:
    lottie-ios (from `../node_modules/lottie-ios`)

    lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.3, which depends on
      lottie-ios (~> 3.2.3)
```

I've already run `pod install --repo-update` and removed `Podfile.lock`.
Or should I wait ` lottie-react-native` to not depend on` lottie-ios (~> 3.2.3)`?

It works fine whilst using ` lottie-ios (^3.2.3)`

**Comment by calda** [comment]
It looks like `lottie-react-native` depends on Lottie 3.4.0 as of this PR: https://github.com/lottie-react-native/lottie-react-native/pull/900

**Comment by devanshsadhotra25** [comment]
> @calda I am having the same issue here. `[!] CocoaPods could not find compatible versions for pod "lottie-ios": In snapshot (Podfile.lock): lottie-ios (from`../node_modules/lottie-ios`)
> 
> In Podfile: lottie-ios (from `../node_modules/lottie-ios`)
> 
> ```
> lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.3, which depends on
>   lottie-ios (~> 3.2.3) `
> ```

+ 1 for this

**Comment by calda** [comment]
This seems like an issue with [lottie-react-native](https://github.com/lottie-react-native/lottie-react-native)'s `podfile`, please file an issue on their repo.
